src/AppBundle/Controller/EvolutionController.php

```diff
@@ -67,6 +67,12 @@ public function updateEvolutionAction(Request $request)
         if (($evolution = $evolutionManager->get($id)) === null) {
             throw new NotFoundHttpException('project.feedback.not_found');
         }
+
+        if (!$this->getUser()->hasRole('ROLE_DEVELOPER') &&
```

Comment:
Careful ! The wanted logic is:

* To update the feedback status, you must be a developer.
* To update the feedback description, you must be the feedback author.

The way you wrote it, the rule is: *to update the feedback description or its status, you must be the feedback author or a developer*. Which means that a developer can update a feedback he did not write, and that an author can update the status of one of its feedbacks.